modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,435 @@
+----
+
+And you use Spark as a processing engine, configured to use the `streaming` catalog:
+
+```
+spark.sql.catalog.streaming = org.apache.iceberg.spark.SparkCatalog
+spark.sql.catalog.streaming.type = rest
+spark.sql.catalog.streaming.uri = http://catalog-service:8181
+```
+
+Using Spark SQL, you can query the Iceberg table directly by specifying the catalog name:
+
+[,sql]
+----
+SELECT * FROM streaming.redpanda.ClickEvent;
+----
+
+Spark can use the REST catalog to automatically discover the topic's Iceberg table. 
+
+==== Filesystem-based catalog (`object_storage`)
+
+If using the `object_storage` catalog type, you must set up the catalog integration in your processing engine accordingly. For example, you can configure Spark to use a filesystem-based catalog with at least the following properties:
+
+```
+spark.sql.catalog.streaming.type = hadoop
+spark.sql.catalog.hadoop_prod.warehouse = s3a://<bucket-name>/path/to/redpanda-iceberg-table
+```
+
+Depending on your processing engine, you may also need to create a new table for the Iceberg data.
+
+== Access data in Iceberg tables
+
+You can use the same analytical tools to access table data in a data lake as you would for a relational database.
+
+For example, suppose you produce the same stream of events to a topic `ClickEvent`, which uses a schema, and another topic `ClickEvent_key_value`, which uses the key-value mode. The topics have Tiered Storage configured to an AWS S3 bucket. A sample record contains the following data:
+
+[,bash,role=no-copy]
+----
+{"user_id": 2324, "event_type": "BUTTON_CLICK", "ts": "2024-11-25T20:23:59.380Z"}
+----
+
+When you point your Iceberg-compatible tool or framework to the object storage location of the Iceberg tables, how you consume the data depends on the topic Iceberg mode and whether you've registered a schema on which to derive the table format. Depending on the processing engine, you may need to first create a new table that gets added to your Iceberg catalog implementation.
+
+In either mode, you do not need to rely on complex ETL jobs or pipelines to access real-time data from Redpanda in your data lakehouse.
+
+=== Query topic with schema (`value_schema_id_prefix` mode)
+
+In this example, it is assumed you have created the `ClickEvent` topic and set `redpanda.iceberg.mode` to `value_schema_id_prefix`. The following is an Avro schema for `ClickEvent`:
+
+.`schema.avsc`
+[,avro]
+----
+{
+    "type" : "record",
+    "namespace" : "com.redpanda.examples.avro",
+    "name" : "ClickEvent",
+    "fields" : [
+       { "name": "user_id", "type" : "int" },
+       { "name": "event_type", "type" : "string" },
+       { "name": "ts", "type": "string" }
+    ]
+ }
+----
+
+You can register the schema under the `ClickEvent-value` subject:
+
+[,bash]
+----
+rpk registry schema create ClickEvent-value --schema path/to/schema.avsc --type avro
+----
+
+If you produce to the `ClickEvent` topic in the following format:
```

Comment:
```suggestion
You can then produce to the `ClickEvent` topic using the following format:
```
I'm not sure about this but the lead in sentence doesn't seem flow from the previous step.